components/inputs/input-text.js

```diff
@@ -253,11 +253,16 @@ class InputText extends RtlMixin(LitElement) {
 		this._hovered = false;
 	}
 
-	_onSlotChange() {
+	_onSlotChange(e) {
+		const slotContent = e.target.assignedNodes()[0];
+		if (!slotContent) return;
+		const id = e.target.parentNode.id;
+
 		this.requestUpdate(); // needed for legacy Edge
```

Comment:
Note, `requestUpdate` returns the same promise that the `updateComplete` would in this scenario.  So this could be written like:
```
this.requestUpdate().then(() => {
    ...
});
```
Or...
```
async _onSlotChange(e) {
    ...
    await this.requestUpdate();
    ...
}
```